conda-forge.yml

```diff
@@ -6,3 +9,9 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+test: native_and_emulated
+azure:
+  free_disk_space: true
```

Comment:
As currently written the aforementioned conda-smithy PR, will use this on Windows as well

Reply:
`zig` seems to be very hard to build. On PPC64LE, none of the 4 method work either, native from source, native with zig, cross-cmake, or cross-zig - The cross-cmake fails with relocation errors when trying to link zig2, where this one is failing.
In previous attempts, I saw failures with LLVM: Memory Error, so that's why I wondered about using a machine with large memory
I tried to ask on their `Issue` board but they have not responded

The method they advertise is to build LLVM+CLang+LLD prior to zig, but I of course want to use the conda packages for those